apps/system/js/multi_screen_controller.js

```diff
@@ -193,11 +258,13 @@
       }
 
       this._enabled = value;
-      if (this._enabled) {
-        window.addEventListener('mozChromeEvent', this);
-      } else {
-        window.removeEventListener('mozChromeEvent', this);
+    },
+    '_observe_dom.presentation.discovery.enabled': function(value) {
+      if (value == this._presentationEnabled) {
+        return;
       }
+
+      this._presentationEnabled = value;
```

Comment:
Since `_presentationEnabled` has never been used anywhere, should we check it in `choosePresentationDevice`? Otherwise, we may not need to observe it, right?